modules/olm-deleting-operators-from-a-cluster-using-cli.adoc

```diff
@@ -22,41 +22,41 @@ endif::openshift-dedicated,openshift-rosa[]
 
 .Procedure
 
-. Check the current version of the subscribed Operator (for example, `jaeger`) in the `currentCSV` field:
+. Check the current version of the subscribed Operator (for example, `serverless-operator`) in the `currentCSV` field:
```

Comment:
What is the Operator being checked for?

If the user is just checking that the subscribed Operator is defined, I'd reword the sentence to: "Ensure the latest version of the subscribed operator is identified in the **currentCSV** field."

Reply:
@nalhadef This is a wonderful suggestion. Thank you so much! I have incorporated it.

Reply:
Thanks, @kaldesai!